.github/workflows/gh-pages.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   Documentation:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
Is there a specific reason for this change? If yes, then it could be mentioned in the commit comment.

Reply:
The reasoning was to install a newer version of Doxygen to reveal possible new warnings. I'll update the commit log.